tests/unit/bokeh/models/test_layouts__models.py

```diff
@@ -70,6 +70,25 @@ def test_Column() -> None:
     check_props_with_sizing_mode(Column())
     check_children_prop(Column)
 
+def check_children_prop_boxes(layout_callable: type[HBox | VBox]):
+    ## component subclasses are layouts, widgets and plots
+    components = [Row(), Column(), figure()]
+
+    # Test layout accepts children argument
+    layout2 = layout_callable(children=components)
+    assert layout2.children == [{'child': c} for c in components]
+    assert layout2._check_repeated_layout_children() == []
+    # Test value error raised when non-layout is provided as children
```

Comment:
It would be ideal to also add a duplicate after this, and then verify that the check finds the duplicate